### PR TITLE
Fixes #821: Fixes Broken Links

### LINKED ISSUE #821: Fix Talawa-API broken links visible when starting the Talawa-Docs app
**Describe the bug**
When starting Talawa-Docs there are numerous references to broken links in the console. Most, **but not all,** are related to  `README.md` files

**To Reproduce**

1. Start Talawa-Docs
2. The console shows broken links

**Expected behavior**

1. Start Talawa-Docs
2. No broken links are discovered in startup.
    1. The end user must get no 404 errors when clicking any link in talawa-docs generated websites
4. All other web page content is maintained.

**Actual behavior**

- See above

**Screenshots**

**Additional details**

image

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by ghost**
**What kind of change does this PR introduce?**
This PR fixes the issue of broken links warning shown in console while starting talawa docs 

Fixes #821 

**Screenshot:** 

Screenshot from 2024-05-20 12-09-58
No Warnings in Console 

**If relevant, did you update the documentation?**

Yes

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: docs/internships/gsod/ideas-2024.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-id: gsod-ideas
+id: gsod-ideas-2024
```
Comment (palisadoes) [inline docs/internships/gsod/ideas-2024.md]: Revert this change. This was not a cause of an error.

File: docs/internships/gsod/ideas.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
+---
+id: gsod-ideas
```
Comment (palisadoes) [inline docs/internships/gsod/ideas.md]: Exclude this file from the PR. This was not a cause of an error.

**Review by palisadoes — CHANGES_REQUESTED** [review]

**Review by palisadoes — CHANGES_REQUESTED** [review]
see comments

**Comment by ghost** [comment]
Okay @palisadoes I'll revert the changes, but I did it because there was one more warning in ideas file 


Screenshot from 2024-05-24 07-52-19

**Review by palisadoes — APPROVED** [review]
